app/meteor-accounts-saml/server/saml_server.js

```diff
@@ -92,11 +92,11 @@ Meteor.methods({
 });
 
 Accounts.registerLoginHandler(function(loginRequest) {
-	if (!loginRequest.saml || !loginRequest.credentialToken) {
```

Comment:
This change broke the mobile apps I think (cc @geekgonecrazy)

Reply:
Yup I think so as well...  :(

Reply:
can't the mobile app send the connection id instead of the random secret?